src/classes/AFFL_AccRecordType_TDTM.cls

```diff
@@ -70,17 +70,17 @@ public class AFFL_AccRecordType_TDTM extends TDTM_Runnable {
          //
          //    B. If that account did not have a record type:
          //         --> the key affl lookup field for that type in Contacts affiliated to it needs to be populated
-    	 if(newlist != null && newlist.size() > 0 && oldlist != null && oldlist.size() > 0) {
```

Comment:
Nothing really changes in this class at the moment, except for minor formatting updates.